egs/wsj/s5/utils/combine_ali_dirs.sh

```diff
@@ -1,99 +0,0 @@
-#!/bin/bash
-# Copyright 2016  Xiaohui Zhang  Apache 2.0.
-
-# This srcipt operates on alignment directories, such as exp/tri4a_ali
-
-# Begin configuration section. 
-cmd=run.pl
-extra_files= 
-num_jobs=4
-# End configuration section.
-echo "$0 $@"  # Print the command line for logging
-
-if [ -f path.sh ]; then . ./path.sh; fi
-. parse_options.sh || exit 1;
-
-if [[ $# -lt 3 ]]; then
-  echo "Usage: $0 [options] <data> <dest-ali-dir> <src-ali-dir1> <src-ali-dir2> ..."
-  echo "e.g.: $0 --num-jobs 32 data/train exp/tri3_ali_combined exp/tri3_ali_1 exp_tri3_ali_2"
-  echo "Options:"
```

Comment:
Sorry about that, the file is available now.

On Thu, Jul 21, 2016 at 7:39 PM, Daniel Povey <notifications@github.com>
wrote:

> In egs/wsj/s5/utils/combine_ali_dirs.sh
> <https://github.com/kaldi-asr/kaldi/pull/875#discussion_r71806911>:
>
> > -# This srcipt operates on alignment directories, such as exp/tri4a_ali
> > -
> > -# Begin configuration section.
> > -cmd=run.pl
> > -extra_files=
> > -num_jobs=4
> > -# End configuration section.
> > -echo "$0 $@"  # Print the command line for logging
> > -
> > -if [ -f path.sh ]; then . ./path.sh; fi
> > -. parse_options.sh || exit 1;
> > -
> > -if [[ $# -lt 3 ]]; then
> > -  echo "Usage: $0 [options] <data> <dest-ali-dir> <src-ali-dir1> <src-ali-dir2> ..."
> > -  echo "e.g.: $0 --num-jobs 32 data/train exp/tri3_ali_combined exp/tri3_ali_1 exp_tri3_ali_2"
> > -  echo "Options:"
>
> I think you forgot to 'git add' this in its location in steps.
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/kaldi-asr/kaldi/pull/875/files/c9298ce792414a31f45da61a734afd38c77426f5#r71806911>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/ADtwoNZe9fTJ6BFvMHxqX06t2JqKZT6Vks5qYANMgaJpZM4JCR8D>
> .
>